src/app/app.module.ts

```diff
@@ -18,6 +19,7 @@ import { GroceryModule } from 'src/app/grocery/grocery.module';
   ],
   declarations: [
     AppComponent,
+    NavigationComponent,
```

Comment:
It should be better to create core module or app layout module, or smth like this and move navigation component there